src/frame/worker/crypto/recrypt/tests/RecryptWasm.test.ts

```diff
@@ -52,6 +52,48 @@ describe("RecryptWasm", () => {
         });
     });
 
+    describe("rotateGroupPrivateKeyWithRetry", () => {
+        const groupPrivateKey = new Uint8Array([22, 33, 44]);
+        it("should result in an error when mocked generateKeyPair returns Uint8Array of zeros for new privateKey", () => {
+            jest.spyOn(Recrypt.getApi(), "generatePlaintext");
```

Comment:
Should this spy be returning something? I looks like this test is meant to verify that if the plaintext we generate is all zeros it should fail, but it looks like here we're not generating anything? It should probably have a `.mockReturnValue(new Uint8Array(32))`.

Reply:
In the rycrypt-wasm-binding  generatePlaintext is mocked for jest or other unit to return new Uint8Array(384). Do I still need to mock it here?

Reply:
Oh right I forgot about the mock here. But when you re-mock it like you're doing here, it will override the other one. So I'm pretty sure it's not returning anything here. We should make it be explicit and have it return what you want based on what you're testing.

Reply:
ok, done